include/apps/watchfaces/OswAppWatchfaceNumerals.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <osw_hal.h>
+#include <osw_ui.h>
+
+#include "osw_app.h"
+#ifdef ANIMATION
+#include <animations/anim_matrix.h>
+#endif
+
+class OswAppWatchfaceNumerals : public OswApp {
+  public:
+    OswAppWatchfaceNumerals(void) {
+        ui = OswUI::getInstance();
+    };
+    virtual void setup() override;
+    virtual void loop() override;
+    virtual void stop() override;
+    ~OswAppWatchfaceNumerals() {};
+
+#ifdef OSW_FEATURE_STATS_STEPS
+    static void drawStepHistory(OswUI* ui, uint8_t x, uint8_t y, uint8_t w, uint8_t h, uint32_t max);
+#endif
+  private:
+    OswUI* ui;
+    void drawWatch();
+    void drawNShiftedTicks(Graphics2D* gfx, uint8_t cx, uint8_t cy, uint8_t r1, uint8_t r2, uint8_t nTicks, float shift, uint16_t color);
+    void drawNShiftedMaskedTicks(Graphics2D* gfx, uint8_t cx, uint8_t cy, uint8_t r1, uint8_t r2, uint8_t nTicks, float shift, uint16_t m, uint16_t color);
```

Comment:
not used :)

Reply:
Indeed, removed:)

Reply:
Yeah i see 👍 
I mean, please delete this code.